src/manifest.json

```diff
@@ -9,7 +9,7 @@
             "matches": [
                 "<all_urls>"
             ],
-            "js": ["js/compatibility.js","js/pdf.js","js/pdf.worker.js","js/pdftotext.js","js/bundle.js", "js/blacklist2.js","js/background.js","js/lib/stopwords_res.js", "js/lib/readability.js", "js/relevanttext.js", "js/content.js"]
```

Comment:
what do you mean its no longer there? 
Do you build the extension before installing it in the browser?